### ISSUE TITLE
Battery Capacity overflow Issue Exceeding 32-bit Limit

### ISSUE THREAD

**Issue by INV-PLA** [mrh0/createaddition#834]
When the total battery capacity exceeds 32 bits, the capacity overflows and becomes negative. If possible, please increase the upper limit of the value.

**Comment by mrh0** [comment]
Yeah, honestly don't know if it is, there might be a workaround but thats decided by Forge, not sure how other mods do it
